git-publish

```diff
@@ -359,18 +366,26 @@ def git_save_email_lists(topic, to, cc, override_cc):
     if not override_cc:
         git_set_config('branch', topic, 'gitpublishcc', cc)
 
-def inspect_menu(tmpdir, to_list, cc_list, in_reply_to,
+def inspect_menu(tmpdir, to_list, cc_list, suppress_cc, in_reply_to,
                  topic, override_cc):
     while True:
         print 'Stopping so you can inspect the patch emails:'
         print '  cd %s' % tmpdir
         print
+        output = git_send_email(to_list, cc_list, tmpdir, suppress_cc, in_reply_to, dry_run=True)
+        index = 0
         files = os.listdir(tmpdir)
         files.sort()
         for f in files:
             tf = open(os.path.join(tmpdir, f), 'rb')
             m = message_from_file(tf)
             print parse_header(m['subject'])
+            # Print relevant 'Adding cc' lines from the git-send-email --dry-run output
+            while index < len(output) and len(output[index]):
+                if output[index].find('Adding cc') != -1:
+                    print '  ' + output[index]
+                index += 1
+            index += 1
```

Comment:
We rely on git-send-email printing the emails in the right order and on separating them with an empty line. Not pretty, but should be fine. Btw, message_from_file/message_from_string won't parse the output due to the extra "non-header" output.